app/views/galleries/_expandable.haml

```diff
@@ -3,7 +3,7 @@
   %td.gallery-name{class: klass}
     = link_to gallery.name, gallery_path(gallery)
     .tag-box
-      - gallery.gallery_groups_data.each do |group|
+      - gallery.gallery_groups_data.ordered_by_gallery_tag.each do |group|
```

Comment:
I think this is invalid if we use a subquery to get this data (since we use an array of fake things), so I'll look into this a bit more.

Reply:
I have no idea but it does in fact need to be ordered, it wasn't.